package.json

```diff
@@ -107,12 +108,13 @@
         "redux": "4.2.1",
         "redux-devtools-extension": "2.13.9",
         "redux-thunk": "2.4.2",
+        "remove": "^0.1.5",
```

Comment:
This seems to have been added by accident?
I can't find any usage from it in the code.